website_user_login_redirect_custom/__manifest__.py

```
@@ -0,0 +1,19 @@
+    "category": "Website",
+    "author": "Cetmix OÜ, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/website",
+    "depends": ["website"],
+    "data": [
+        "data/ir_config_parameter_data.xml",
+        "views/res_config_settings_views.xml",
+    ],
+    "installable": True,
+    "application": False,
+}
```

Pylint (4.0.0)

[warning] Statement seems to have no effect

(W0104)